api/pom.xml

```diff
@@ -39,6 +39,16 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+		 <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+            <version>3.0.1</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>jsp-api</artifactId>
+        </dependency>
```

Comment:
These shouldn't be in the api module. Things need to be refactored so that all the stuff that depends on servlet classes is done in the `web` project.